### ISSUE TITLE
[WinMSR]: "ValidateNTTargetVersion" was not assigned a value for the mandatory parameter "ValidNTTargetVersions".

### ISSUE THREAD

**Issue by MatteoBax** [intel/pcm#615]
When I try to compile the MSR driver for Windows from Visual Studio I get this:
````
Project "D:\Users\Matteo\source\repos\pcm\src\WinMSRDriver\MSR.vcxproj" on node 1 (default targets).
D:\Windows Kits\10\build\10.0.22621.0\WindowsDriver.common.targets(314,5): error MSB4044: Task "ValidateNTTargetVersion" was not assigned a value for the mandatory parameter "ValidNTTargetVersions". [D:\Users\Matteo\source\repos\pcm\src\WinMSRDriver\MSR.vcxproj]
````

**Comment by opcm** [comment]
thank you for reporting this. I am not able to reproduce the issue on my Windows machine. No error..

**Comment by jjasoliya** [comment]
My build fails with following error.

```
(base) PS C:\Users\jaymi\source\repos\Intel_PCM\pcm\src> cd .\WinMSRDriver\
(base) PS C:\Users\jaymi\source\repos\Intel_PCM\pcm\src\WinMSRDriver> MSBuild.exe MSR.vcxproj -property:Configuration=Release -property:Platform=x64
MSBuild version 17.8.3+195e7f5a3 for .NET Framework
Build started 1/10/2024 3:36:07 PM.

Project "C:\Users\jaymi\source\repos\Intel_PCM\pcm\src\WinMSRDriver\MSR.vcxproj" on node 1 (default targets).
C:\Program Files (x86)\Windows Kits\10\build\10.0.22621.0\WindowsDriver.common.targets(314,5): error MSB4044: The "ValidateNTTargetVersion" task was not given a value for the required parameter "ValidNTTargetVersions". [C:\Users\jaymi\source\repos\Intel_PCM\pcm\src\WinMSRDriver\MSR.vcxproj]
Done Building Project "C:\Users\jaymi\source\repos\Intel_PCM\pcm\src\WinMSRDriver\MSR.vcxproj" (default targets) -- FAILED.


Build FAILED.

"C:\Users\jaymi\source\repos\Intel_PCM\pcm\src\WinMSRDriver\MSR.vcxproj" (default target) (1) ->
(ValidateNTTargetVersion target) ->
  C:\Program Files (x86)\Windows Kits\10\build\10.0.22621.0\WindowsDriver.common.targets(314,5): error MSB4044: The "ValidateNTTargetVersion" task was not given a value for the required parameter "ValidNTTargetVersions". [C:\Users\jaymi\source\repos\Intel_PCM\pcm\src\WinMSRDriver\MSR.vcxproj]

    0 Warning(s)
    1 Error(s)
```

**Comment by rdementi** [comment]
this issue should now be resolved in the master branch : https://github.com/intel/pcm/commit/56901851deda8a6f78254fa9fbef3d4c3afdf7a4

**Comment by rdementi** [comment]
please reopen if the issue is still there

**Comment by MatteoBax** [comment]
> this issue should now be resolved in the master branch : [5690185](https://github.com/intel/pcm/commit/56901851deda8a6f78254fa9fbef3d4c3afdf7a4)

The issue has been solved, thanks.
